docs/book/v4/installation/faq.md

```diff
@@ -0,0 +1,33 @@
+# Frequently Asked Questions
+
+## How do I fix common permission issues?
+
+If running your project you encounter some permission issues, follow the below steps.
+
+### Errors:
+
+> PHP Fatal error:  Uncaught InvalidArgumentException: The directory "/var/www/_example.local_/html/data" is not writable...
+
+> PHP Fatal error:  Uncaught InvalidArgumentException: The directory "/var/www/_example.local_/html/data/cache" is not writable...
+
+> PHP Fatal error:  Uncaught InvalidArgumentException: The directory "/var/www/_example.local_/html/data/cache/doctrine" is not writable...
+
+**Fix:**
+
+    chmod -R 777 data
+
+### Error:
+
+> PHP Fatal error:  Uncaught InvalidArgumentException: The directory "/var/www/_example.local_/html/public/uploads" is not writable...
+
+**Fix:**
+
+    chmod -R 777 public/uploads
```

Comment:
```suggestion
```shell
chmod -R 777 public/uploads
```
Mind the closing backticks after the command (```) - Github does not let me add them directly in the suggestion.